crawler/dockercontainer.py

```diff
@@ -110,8 +109,11 @@ def get_memory_cgroup_path(self, node='memory.stat'):
                             self.long_id, node)
 
     def get_cpu_cgroup_path(self, node='cpuacct.usage'):
-        return os.path.join(self.get_cgroup_dir('cpuacct'), 'docker',
-                            self.long_id, node)
+        cgroup_dir = self.get_cgroup_dir('cpuacct')
+        if not cgroup_dir:
+            # In kernels 4.x, the node is actually called 'cpu,cpuacct'
```

Comment:
do you have a doc or a system I can poke at to check this? I have no way of confirming this otherwise.